src/lib/deploy-helpers.ts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { K8s, kind, KubernetesListObject } from "kubernetes-fluent-client";
+import Log from "./logger";
+
+// get all deployments
+export async function getDeployments() {
+  const deployments = await K8s(kind.Deployment).InNamespace("pepr-system").Get();
+  return deployments;
+}
+
+// waits for all replicas to be ready for all deployments
+export async function waitForAllDeploymentReplicas(deployments: KubernetesListObject<kind.Deployment>) {
+  let status = false;
+  let readyCount = 0;
+
+  if (deployments.items.length === 0) {
+    Log.info("No deployments found");
+    return status;
+  }
+
+  for (const deployment of deployments.items) {
+    Log.info(`${deployment.metadata?.namespace}/${deployment.metadata?.name} is deploying`);
+
+    while (deployment.status?.readyReplicas !== deployment.spec?.replicas) {
+      const readyReplicas = deployment.status?.readyReplicas ? deployment.status?.readyReplicas : 0;
+      Log.info(
+        `${deployment.metadata?.namespace}/${deployment.metadata?.name} is not ready - ${readyReplicas} ready replicas out of ${deployment.spec?.replicas} spec replicas`,
```

Comment:
Remove the namespace since we are only concerned about `pepr-system`.

Lets use this style log (output from `kubectl rollout status deploy/<name>`)
```
Waiting for deployment "<deploy-name>" rollout to finish: 17 of 23 updated replicas are available...
```

Reply:
Namespace removed and log style updated